attributes/mesos_options.rb

```diff
@@ -380,12 +380,6 @@
       'flag' => false,
       'deprecated' => false
     },
-    'logs_dir' => {
```

Comment:
Kind of. The mesososphere init wrapper will read it and set a flag for mesos. However, if you set the MESOS_log_dir (which is what the cookbook template does), it will override it as ["Configuration values are searched for first in the environment, then on the command-line."](http://mesos.apache.org/documentation/latest/configuration/)